frontend/server/src/main/java/org/pytorch/serve/http/api/rest/ManagementRequestHandler.java

```diff
@@ -93,7 +94,7 @@ public void handleRequest(
                     } else {
                         handleScaleModel(ctx, decoder, segments[2], modelVersion);
                     }
-                } else if (HttpMethod.DELETE.equals(method)) {
+                } else if (HttpMethod.DELETE.equals(method) && isModeExplicit()) {
```

Comment:
We'll need to include this check for gRPC as well
https://github.com/pytorch/serve/blob/dd1f8dc36efe21f036be1e65043310dec74af28e/frontend/server/src/main/java/org/pytorch/serve/grpcimpl/ManagementImpl.java#L196

Reply:
Created issue for gRPC

Reply:
https://github.com/pytorch/serve/issues/3180